tests/unit/setup-context-test.js

```diff
@@ -862,6 +859,11 @@ module('setupContext', function (hooks) {
       setApplication(isolatedApp);
     });
 
+    hooks.afterEach(function () {
+      delete isolatedApp.constructor.initializers.foo;
```

Comment:
let's remove this (in favor of adding the `class extends App {}` thing I mentioned in the other comment)